conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -9,11 +9,37 @@
 from fastapi import APIRouter, Body, Depends, HTTPException, Query, Request
 from fastapi.responses import PlainTextResponse, RedirectResponse
 
-router_api = APIRouter(
+_API_PREFIX_V1 = "/api/v1"
+_API_PREFIX_V2 = "/api/v2"
+_API_PREFIX_EXPERIMENTAL = "/api/experimental"
+
+router_api_v1 = APIRouter(
+    tags=["api"],
+    prefix=_API_PREFIX_V1,
+)
+
+router_api_v2 = APIRouter(
     tags=["api"],
-    prefix="/api/v1",
+    prefix=_API_PREFIX_V2,
 )
 
+router_api_experimental = APIRouter(
+    tags=["api"],
+    prefix=_API_PREFIX_EXPERIMENTAL,
+)
+
+
+def _is_api_v1(request: Request) -> bool:
+    return request.scope["route"].path.startswith(_API_PREFIX_V1)
+
+
+def _is_api_v2(request: Request) -> bool:
+    return request.scope["route"].path.startswith(_API_PREFIX_V2)
+
+
+def _is_api_experimental(request: Request) -> bool:
+    return request.scope["route"].path.startswith(_API_PREFIX_EXPERIMENTAL)
```

Comment:
These functions are not currently used, but I've added them while I'm making changes to routing, since they are useful. They allow determining, from the request handler function body, which API version the client is calling. This can be used to implement API-version-specific logic.